tools/bbbfancontrol/Makefile

```diff
@@ -1,6 +1,9 @@
-.DEFAULT_GOAL := native
+.DEFAULT_GOAL := aarch64
+REPO_ROOT=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))/../..
 
-native:
-	go build
-aarch64:
-	GOARCH=arm64 go build
+native: copy-service
+	go build -o $(REPO_ROOT)/build/bbbfancontrol
```

Comment:
If run by itself, this Makefile aborts due to the missing `build` directory. Can we create it if necessary?

Reply:
> If run by itself, this Makefile aborts due to the missing build directory. Can we create it if necessary?

Yeah, I was aware of that trade-off, but imagined that a user would run the top-level `make` at least once, which would have created the `build/`.

I can for sure add a `mkdir -p $(REPO_ROOT)/build` here, but then we also should have it in `tools/Makefile`, and it might get a bit repetitive..

Let me know your preference.